src/plugins/analysis/hardware_analysis/view/hardware_analysis.html

```diff
@@ -0,0 +1,22 @@
+{% extends "analysis_plugins/general_information.html" %}
+
+{% block analysis_result_details %}
+
+
+<tr>
+	<td class="table-head-light"><b>Plug-in</b></td>
+	<td class="table-head-light"><b>Detection</b></td>
+</tr>
+	{% for key, value in firmware.processed_analysis[selected_analysis].items() %}
+		{% if value %}
+			{% if key | is_not_mandatory_analysis_entry %}
+				<tr>
+					<td> {{ key }} </td>
+					{% set entry = "\n".join(value) if value is iterable else value %}
+					<td><pre><code style="word-break: break-all; white-space: pre-wrap;">{{ firmware.processed_analysis[selected_analysis][key] | nice_generic | safe }}</code></pre></td>
```

Comment:
The `join()` isn't really necessary, since the `nice_generic` filter seems to handle lists just fine, so we could remove it
```suggestion
					<td><pre><code style="word-break: break-all; white-space: pre-wrap;">{{ firmware.processed_analysis[selected_analysis][key] | nice_generic | safe }}</code></pre></td>
```